### DISCUSSION TITLE
Allure Server is Unreachable: Get "": unsupported protocol scheme ""

### DISCUSSION THREAD

**Discussion by cleandust** [allure-framework/allure2#2172] [Questions & Support] [answered]
Hi, I am providing `ALLURE_TOKEN`, `ALLURE_ENDPOINT` and `ALLURE_PROJECT_ID` but when trying to upload test results to Allure TestOps, It keeps exiting with:

`Allure Server is Unreachable: Get "": unsupported protocol scheme ""`

Any ideas ?

**Comment by cleandust** [comment]
Allure TestOps Client v2.2.1, this is the version i am using if it helps

**Reply by cheshi-mantu** [reply]
Hi @cleandust 
would you care to show exact commands you are using 
- to get the binary
- set up the required parameters for the connection
- full command you are using

cheers

**Reply by cleandust** [reply]
Hi @cheshi-mantu, let me briefly explain what exactly I am doing, should have done it at the beginning anyways )))

 1. Running UI tests by executing `./gradlew`
 2. Then pulling the file where test results are saved through adb: `adb  pull sdcard/Download/testresults`
 3. After pulling the file, running `./allurectl upload testresults` which gives me the above-mentioned error

Doubled checked the credentials, I am providing they look correct.

**Reply by cleandust** [reply]
`set up the required parameters for the connection` => what exactly you mean by this ? As I understand, providing these env vars suffice.

**Reply by cheshi-mantu** [reply, marked as answer]
that was exactly my question
we need to understand the complete list of actions, see the example here: https://docs.qameta.io/allure-testops/ecosystem/allurectl/#allurectl-watch-workflow
